Dockerfile

```diff
@@ -43,12 +34,10 @@ RUN cp ./ocm-backplane /out
 RUN chmod -R +x /out
 
 ### Build dependencies
-FROM registry.access.redhat.com/ubi9/ubi:9.5 as dep-builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.23 as dep-builder
 
-RUN yum install --assumeyes \
-    jq \
-    unzip \
-    wget
+RUN dnf update -y && \
+dnf install -y jq 
```

Comment:
```suggestion
    RUN dnf update -y && \
    dnf install -y curl && \
    curl -LO https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && \
    chmod +x jq-linux64 && \
    mv jq-linux64 /usr/bin/jq
```

Reply:
This might help in with the build fixture, as currently, rhel_9 doesn't contain yum for installing jq.